jupyterhub/config/jupyterhub_config.py

```diff
@@ -152,6 +157,31 @@ class AppAuthenticatorLogoutHandler(LogoutHandler):
     # from openhexa will send us here, after jupyterhub logout the
     # the user, this endpoint will redirect to the login page of openhexa
     # TODO: use API https://github.com/jupyterhub/jupyterhub/issues/3688)
+
+    async def get(self):
+        # Login the user using the sessionid and csrftoken in the cookies
+        user = await self.login_user({})
```

Comment:
I think it's fine but `login_user` does a lot of things (re-attempting to set cookies, logging stuff, marking the user as "reauthenticated"...)

It might be "leaner" to do something like:

```python
authenticated = await self.authenticate(data)
if authenticated:
    user = await self.auth_to_user(authenticated)
else:
    raise ValueError("oopsie")
```

Reply:
Voilà, I've updated to only get the user based on the authenticated data.